Snippets/ASB/ASB_6/Usage.cs

```diff
@@ -13,6 +13,17 @@ class Usage
         #endregion
     }
 
+    void NamespaceRoutingSendOptions(IBus bus)
+    {
+        string destination;
+        #region namespace_routing_send_options_full_connectionstring
+
+        destination = "sales@Endpoint=sb://destination1.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]";
+        bus.Send(destination, new MyMessage());
```

Comment:
I would limit it to 1, there are many places where you can specify an address or destination, like message endpoint mappings, unicast routing api etc... we don't want to show them all